notebooks/samples/AzureSearchIndex - Met Artworks.ipynb

```diff
@@ -17,14 +17,31 @@
     {
       "cell_type": "code",
       "source": [
-        "import os, sys, time, json, requests\nfrom pyspark.ml import Transformer, Estimator, Pipeline\nfrom pyspark.ml.feature import SQLTransformer\nfrom pyspark.sql.functions import lit, udf, col, split"
+        "import os, sys, time, json, requests\n",
+        "from pyspark.ml import Transformer, Estimator, Pipeline\n",
+        "from pyspark.ml.feature import SQLTransformer\n",
+        "from pyspark.sql.functions import lit, udf, col, split"
       ],
       "metadata": {
         "collapsed": true
       },
       "outputs": [],
       "execution_count": 3
     },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "if os.environ.get(\"AZURE_SERVICE\", None) == \"Microsoft.ProjectArcadia\":\n",
+        "    from pyspark.sql import SparkSession\n",
+        "    spark = SparkSession.builder.getOrCreate()\n",
+        "    from notebookutils.mssparkutils.credentials import getSecret\n",
+        "    os.environ['VISION_API_KEY'] = getSecret(\"wenqxkv\", \"mmlspark-cs\")\n",
```

Comment:
for the camera-ready code lets use the existing keyvault and move all resources to mmlspark RG

Reply:
done.